docs/overview.asciidoc

```diff
@@ -12,6 +12,11 @@ With Elastic Cloud on Kubernetes you can streamline all those critical operation
 . Securing clusters with TLS certificates
 . Setting up hot-warm-cold architectures with availability zone awareness
 
+Supported versions:
+
+. Kubernetes: 1.11+
+. Elasticsearch: 6.8, 7.1+
```

Comment:
nit: maybe `6.8.x` since there already are some revisions (6.8.3) but 7.0 is not supported.

Reply:
Fixed